spec/rmagick/image/virtual_pixel_methods_spec.rb

```diff
@@ -0,0 +1,25 @@
+RSpec.describe Magick::Image, '#virtual_pixel_method' do
```

Comment:
File name should be `virtual_pixel_method_spec.rb` instead

Reply:
Ha, good catch! Updated. There's a rubocop rule that will hopefully get enabled at some point to make sure the file name matches the method listed in the description.